lua/nomadsunits.lua

```diff
@@ -341,6 +341,22 @@ NExperimentalHoverLandUnit = Class(NHoverLandUnit) {
     end,
 }
 
+SlowHoverLandUnit = Class(HoverLandUnit) {
```

Comment:
no need to add this function (its in defaultunits, i.e. for all factions) for nomads specifically again.